docs/tutorial/spark-tutorial/t-spark-monitoring-with-cos.md

```diff
@@ -0,0 +1,76 @@
+### Spark Monitoring with Canonical Observability Stack
+
+
+The Charmed Spark solution comes with the [spark-metrics](https://github.com/banzaicloud/spark-metrics) exporter embedded in the [Charmed Spark OCI image](https://github.com/canonical/charmed-spark-rock), used as base for driver and executors pods .
+This exporter is designed to push metrics to the [prometheus pushgateway](https://github.com/prometheus/pushgateway), that is integrated with the [Canonical Observability Stack](https://charmhub.io/topics/canonical-observability-stack). 
```

Comment:
Uhm, the pushgatewas seems missing in this page

Reply:
I guess it was a temporary problem, because the link is working now.